src/lib/types.ts

```diff
@@ -128,6 +129,10 @@ export enum QueryOperation {
   IS_GREATER_THAN_OR_EQUAL = '$gte',
   EXISTS = '$exists',
   MATCHES = '$regex',
+  ABOVE = '$above',
```

Comment:
Can we seperate these operators to a different enum since these are not available for entry query or asset query? @abhinav-from-contentstack

Reply:
done